tests/PHPStan/Rules/Functions/data/printf.php

```diff
@@ -6,10 +6,10 @@
 sprintf('%s %s', 'foo'); // one parameter missing
 sprintf('foo', 'foo'); // one parameter over
 sprintf('foo %s', 'foo', 'bar'); // one parameter over
-sprintf('%2$s %1$s %% %1$s %%%', 'one'); // one parameter missing
+sprintf('%2$s %1$s %% %1$s %%', 'one'); // one parameter missing
 sprintf('%2$s %%'); // two parameters required
 sprintf('%2$s %1$s %1$s %s %s %s %s'); // four parameters required
-sprintf('%2$s %1$s %% %s %s %s %s %%% %%%%', 'one', 'two', 'three', 'four'); // ok
```

Comment:
This was producing an error https://3v4l.org/2dPI6

Reply:
we had such test problems in the past. I wonder whether we can somehow detect such bogus tests
(in the general case, not just in (s)printf)